src/main/kotlin/no/nav/syfo/dialogmote/DialogmoteService.kt

```diff
@@ -401,8 +395,9 @@ class DialogmoteService(
         )
 
         val now = LocalDateTime.now()
+        val isDialogmoteTidPassed = moteTidspunkt.isBefore(now)
```

Comment:
Med disse tilpasningene sendes det verken varsel til arbeidstaker eller til arbeidsgiver for møter som er tilbake i tid (teoretisk sett kan man komme hit også for innkalling og endring av tid/sted)